common/src/main/java/com/wynntils/models/territories/type/GuildResourceValues.java

```diff
@@ -1,12 +1,13 @@
 /*
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
  * This file is released under LGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.models.territories.type;
 
 import net.minecraft.ChatFormatting;
 
 public enum GuildResourceValues {
+    NONE("None", ChatFormatting.WHITE, 0),
```

Comment:
Does the API actually return "NONE", or where does this get set?

Reply:
This is from the advancements menu I believe, which displays "None" as seen in the image I posted above

Reply:
Ah!